.github/ISSUE_TEMPLATE.md

```diff
@@ -1,21 +0,0 @@
-### General Type of Issue:
-Some example types are:
-- bug
-- enhancement
-- question
-
-### Details of Issue:
-(Be sure to give enough details that the issue can be understood sufficiently 
-to replicate or to make the required changes)
-
-### For bugs fill out the following: (Otherwise remove, and remove any prompts that aren't useful)
------------------------------------------------------------------
-#### Important Details of your setup/configuration (compset, settings, etc.):
-
-#### Important output or errors that show the problem:
-
-#### What is the CTSM version you are using (output of git describe)?:
-
-#### What is the machine you are running on:?
-
------------------------------------------------------------------
```

Comment:
My understanding is that you will still need a default issue template? Should that be a copy of `04_other_...`

Reply:
In our case, if the issue doesn't fall under one of the given categories, then we don't have any particular issue template – a blank slate is sufficient.

If you'd like to see how this plays out in practice, I've set up a demo repo where you can open an issue:

https://github.com/billsacks/ctsm-github-multiple-issue-templates/issues

Reply:
oooo, this demo is great. I think you can ignore all my comments on this PR now.